diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/grace/wdi/try
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+GitHub is a plateform that can post and share your repo with other users. It also has more features than Git.
```

Comment:
The first part is true, but git actually has many features that GitHub currently doesn't not support (e.g., staging hunks).